script.js

```diff
@@ -95,8 +95,8 @@ window.onload = () => {
   function drawO(tile, ctx, next) {
     tile.style.backgroundColor = 'orange';
     ctx.beginPath();
-    ctx.arc(100, 100, 50, 0, 2 * Math.PI);
-    ctx.lineWidth = 20;
+    ctx.arc(120, 80, 40, 0, 2 * Math.PI);
+    ctx.lineWidth = 25;
```

Comment:
Hello, @TheJSGirl have you tested this across devices?

Reply:
@ashokdey I have tested it for multiple screens. Please have a look at the attached document.
![tic-tac-toe](https://user-images.githubusercontent.com/20318776/80301661-f6aa9100-87c2-11ea-972e-75cbcb74ba42.png)

Reply:
It's looking great but seems like removing the padding from the container box (one with the black border) will make it even better